docs/BMI-Installation.md

```diff
@@ -0,0 +1,287 @@
+cd tgt
+
+yum install -y make
+yum install -y gcc
+** delete install doc from Makefile if problem occurs in the next three steps
+make CEPH_RBD=1 clean
+make CEPH_RBD=1
+make CEPH_RBD=1 install
+
+
+cp scripts/tgtd.service /usr/lib/systemd/system/
+systemctl daemon-reload
+systemctl list-unit-files | grep -i tgt
+
+iptables -I INPUT -p tcp -m tcp --dport 3260 -j ACCEPT
+service iptables save
+
+systemctl start tgtd
+chkconfig tgtd on
+systemctl status tgtd
+```
+
+For more information visit their [website](http://stgt.sourceforge.net/) and the [Quick Start Guide](https://fedoraproject.org/wiki/Scsi-target-utils_Quickstart_Guide)
+
+#### DHCP server
+We support `dnsmasq` for this. 
+
+* On ubuntu:
+`$ sudo apt-get install dnsmasq`
+
+* On CentOS:
+`$ yum install dnsmasq`
+
+For more information about dnsmasq, you can look at the [wiki](https://wiki.debian.org/HowTo/dnsmasq)
+
+#### TFTP Server (Optional)
+
+dnsmasq has a tftp server bundled which you can enable through it's config file.
+But if you are using some other DHCP service, or want to explicitly install TFTP you can do the following:
+
+##### On ubuntu:
+* Install these packages:  `$ sudo apt-get install xinetd tftpd tftp`
+
+* Create `/etc/xinetd.d/tftp` and put this entry  
+  
+```
+service tftp
+{
+protocol        = udp
+port            = 69
+socket_type     = dgram
+wait            = yes
+user            = nobody
+server          = /usr/sbin/in.tftpd
+server_args     = /tftpboot
+disable         = no
+}
+```
+##### On CentOS:
+* Install these packages:  `$ sudo yum install tftp tftp-server xinetd`
+* Modify `/etc/xinetd/tftp` as shown below  
+
+```
+service tftp
+{
+        socket_type             = dgram
+        protocol                = udp
+        wait                    = yes
+        user                    = root
+        server                  = /usr/sbin/in.tftpd
+        server_args             = /tftpboot
+        disable                 = no
+        per_source              = 11
+        cps                     = 100 2
+        flags                   = IPv4
+}
+```
+
+* Create a folder `/tftpboot` this should match whatever you gave in `server_args.` mostly it will be `tftpboot`:
+  
+```
+$ sudo mkdir /tftpboot
+$ sudo chmod -R 777 /tftpboot
+$ sudo chown -R nobody /tftpboot
+```
+  
+* Restart the `xinetd` service:  `$ sudo service xinetd restart`
+
+
+
+The TFTP server should be up and running. For more information click [here](http://askubuntu.com/questions/201505/how-do-i-install-and-run-a-tftp-server)
+
+#### Hardware Isolation Layer (HIL)
+
+To setup HIL, you can read the [HIL documentation](http://hil.readthedocs.io/en/latest/)
+
+### Installling
+* Clone this repository to your home folder and change to `dev` branch
+```
+$ git clone https://github.com/CCI-MOC/ims
+$ git checkout dev
+```
+* Run setup.py and install python-cephlibs
+```
+$ sudo python setup.py install
+$ pip install python-cephlibs
+```
+
+That's it. Installation is done!
+***
+## Running
+
```

Comment:
running bmi. I changed the title and moved this section where it makes more sense.